diff -ruN semantik-0.7.4.orig/wscript semantik-0.7.4/wscript
--- semantik-0.7.4.orig/wscript	2009-10-12 18:26:26.000000000 +0200
+++ semantik-0.7.4/wscript	2011-06-13 23:24:35.490762785 +0200
@@ -247,9 +247,10 @@
 
 	# the Debian packagers compile with --prefix=/usr and set /etc/ld.so.conf accordingly
 	# the rpath is for end users installing into /usr/local/lib only
-	sur = '/usr/local'
-	if Options.options.prefix[:len(sur)] == sur:
-		conf.env.RPATH_NABLAH = '-Wl,--rpath=/usr/local/lib'
+	#sur = '/usr/local'
+	#if Options.options.prefix[:len(sur)] == sur:
+	#	conf.env.RPATH_NABLAH = '-Wl,--rpath=/usr/local/lib'
+	conf.env.RPATH = '/usr/lib64/qt'
 
 def set_options(opt):
 	opt.tool_options('kde4')
