diff -ruN dhex_0.62.orig/Makefile dhex_0.62/Makefile
--- dhex_0.62.orig/Makefile	2011-01-12 13:34:30.000000000 +0100
+++ dhex_0.62/Makefile	2011-01-17 23:05:50.000000000 +0100
@@ -1,11 +1,5 @@
-CC=		gcc
-LDFLAGS=	-L/usr/lib	-L/usr/local/lib  	-L/usr/lib/ncurses	-L/usr/local/lib/ncurses
-CPPFLAGS=	-I/usr/include	-I/usr/local/include	-I/usr/include/ncurses	-I/usr/local/include/ncurses
-CFLAGS=		-O3 -Wall -std=c99 
-#CFLAGS+= -ffunction-sections -fdata-sections
-#LDFLAGS+= --gc-sections 
+
 LIBS=		-lncurses
-DESTDIR=	/usr/local/bin
 
 OFILES=buffers.o input.o output.o machine_type.o main.o menu.o ui.o hexcalc.o search.o gpl.o configfile.o markers.o
 
@@ -15,8 +9,7 @@
 	$(CC) $(LDFLAGS) -o $@ $(OFILES) $(LIBS)
 
 install:all
-	strip dhex
-	cp dhex $(DESTDIR)
+	install dhex $(DESTDIR)/usr/bin
 	
 
 .c.o:
