Index: qt-core/qt-core-4.6.2-r1.ebuild
===================================================================
--- qt-core.orig/qt-core-4.6.2-r1.ebuild
+++ qt-core/qt-core-4.6.2-r1.ebuild
@@ -169,7 +169,7 @@ src_install() {
 	fi
 
 	if use glib; then
-		QCONFIG_DEFINE="$(use glib && echo QT_GLIB)
+		QCONFIG_DEFINE="QT_SHARED $(use glib && echo QT_GLIB)
 				$(use ssl && echo QT_OPENSSL)"
 		install_qconfigs
 	fi