diff -ruN xrdp-0.9.23.1.orig/configure.ac xrdp-0.9.23.1/configure.ac
--- xrdp-0.9.23.1.orig/configure.ac	2023-09-27 19:59:31.000000000 +0200
+++ xrdp-0.9.23.1/configure.ac	2023-10-05 11:38:12.625508282 +0200
@@ -182,13 +182,6 @@
 AX_CFLAGS_WARN_ALL
 AX_APPEND_COMPILE_FLAGS([-Wwrite-strings])
 
-AM_COND_IF([LINUX],
-  [AX_APPEND_COMPILE_FLAGS([-Werror])]) # bsd has warnings that have not been fixed yet
-
-AM_COND_IF([DEVEL_DEBUG],
-  [AX_APPEND_COMPILE_FLAGS([-g -O0])],
-  [AX_APPEND_COMPILE_FLAGS([-O2])])
-
 # Don't fail without working nasm if rfxcodec is not enabled
 if test "x$enable_rfxcodec" != xyes; then
   with_simd=no
