diff -ruN qtiplot-0.9.8.9-origin-2/qtiplot/importOPJ.pro qtiplot-0.9.8.9/qtiplot/importOPJ.pro
--- qtiplot-0.9.8.9-origin-2/qtiplot/importOPJ.pro	2011-11-06 14:56:52.000000000 +0100
+++ qtiplot-0.9.8.9/qtiplot/importOPJ.pro	2012-01-21 21:00:06.000000000 +0100
@@ -7,6 +7,7 @@
 INCLUDEPATH	+= $$QWT_INCLUDEPATH
 INCLUDEPATH += $$QWT3D_INCLUDEPATH
 INCLUDEPATH	+= $$GSL_INCLUDEPATH
+INCLUDEPATH	+= /usr/include/liborigin2
 
 include(src/core/core.pri)
 include(src/lib/libqti.pri)
diff -ruN qtiplot-0.9.8.9-origin-2/qtiplot/src/plugins/OriginPlugin/liborigin1/liborigin1.pri qtiplot-0.9.8.9/qtiplot/src/plugins/OriginPlugin/liborigin1/liborigin1.pri
--- qtiplot-0.9.8.9-origin-2/qtiplot/src/plugins/OriginPlugin/liborigin1/liborigin1.pri	2011-11-06 16:10:08.000000000 +0100
+++ qtiplot-0.9.8.9/qtiplot/src/plugins/OriginPlugin/liborigin1/liborigin1.pri	2012-01-21 21:03:26.000000000 +0100
@@ -2,33 +2,14 @@
 ################# Origin Import (liborigin) ###################
 ###############################################################
 INCLUDEPATH	+=	src/plugins/OriginPlugin/liborigin1 \
-				src/plugins/OriginPlugin/liborigin2 \
+				/usr/include/liborigin2 \
 				src/plugins/OriginPlugin/tree.phi-sci.com \
 				../3rdparty/boost
 
-HEADERS		+=	src/plugins/OriginPlugin/liborigin1/importOPJ.h \
-				src/plugins/OriginPlugin/liborigin2/OriginObj.h \
-				src/plugins/OriginPlugin/liborigin2/OriginFile.h \
-				src/plugins/OriginPlugin/liborigin2/OriginParser.h \
-				src/plugins/OriginPlugin/liborigin2/Origin410Parser.h \
-				src/plugins/OriginPlugin/liborigin2/Origin500Parser.h \
-				src/plugins/OriginPlugin/liborigin2/Origin610Parser.h \
-				src/plugins/OriginPlugin/liborigin2/Origin700Parser.h \
-				src/plugins/OriginPlugin/liborigin2/Origin750Parser.h \
-				src/plugins/OriginPlugin/liborigin2/Origin800Parser.h \
-				src/plugins/OriginPlugin/liborigin2/Origin810Parser.h \
-				src/plugins/OriginPlugin/liborigin2/Origin850Parser.h
+HEADERS		+=	src/plugins/OriginPlugin/liborigin1/importOPJ.h
 
-SOURCES		+=	src/plugins/OriginPlugin/liborigin1/importOPJ.cpp \
-				src/plugins/OriginPlugin/liborigin2/OriginFile.cpp \
-				src/plugins/OriginPlugin/liborigin2/OriginParser.cpp \
-				src/plugins/OriginPlugin/liborigin2/Origin410Parser.cpp \
-				src/plugins/OriginPlugin/liborigin2/Origin500Parser.cpp \
-				src/plugins/OriginPlugin/liborigin2/Origin610Parser.cpp \
-				src/plugins/OriginPlugin/liborigin2/Origin700Parser.cpp \
-				src/plugins/OriginPlugin/liborigin2/Origin750Parser.cpp \
-				src/plugins/OriginPlugin/liborigin2/Origin800Parser.cpp \
-				src/plugins/OriginPlugin/liborigin2/Origin810Parser.cpp \
-				src/plugins/OriginPlugin/liborigin2/Origin850Parser.cpp
+SOURCES		+=	src/plugins/OriginPlugin/liborigin1/importOPJ.cpp
+
+LIBS		+=	-lorigin2
 
 DEFINES		+=	NO_LOG_FILE
