diff -ruN digikam-2.0.0-beta1.orig/extra/libkface/test/CMakeLists.txt digikam-2.0.0-beta1/extra/libkface/test/CMakeLists.txt
--- digikam-2.0.0-beta1.orig/extra/libkface/test/CMakeLists.txt	2011-01-02 16:22:04.000000000 +0100
+++ digikam-2.0.0-beta1/extra/libkface/test/CMakeLists.txt	2011-01-30 23:20:00.000000000 +0100
@@ -42,4 +42,4 @@
 KDE4_ADD_UI_FILES(libfaceGUI_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/gui/mainwindow.ui)
 KDE4_ADD_EXECUTABLE(libfaceGUI ${libfaceGUI_SRCS})
 
-TARGET_LINK_LIBRARIES(libfaceGUI kface ${KDE4_KDECORE_LIBS} ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY})
+TARGET_LINK_LIBRARIES(libfaceGUI kface ${KDE4_KDECORE_LIBS} ${QT_QTCORE_LIBRARY} ${KDE4_KDEUI_LIBS} ${QT_QTGUI_LIBRARY} ${OpenCV_LIBRARIES})
