--- drivers/Makefile.in.original	2008-01-31 05:09:48.000000000 +0900
+++ drivers/Makefile.in	2008-02-18 16:33:48.000000000 +0900
@@ -549,18 +549,16 @@
 	$(MAKE) -C $(LINUX_SRCDIR) V=1 modules_install\
 		CC="$(LINUX_CC) -I@abs_top_srcdir@ -I@abs_top_srcdir@/driver/include -I@abs_top_srcdir@/include" \
 		SUBDIRS="$(GPIB_ABS_MODULE_SUBDIR)" INSTALL_MOD_DIR="gpib"
-	$(DEPMOD) -ae
 
 /dev/gpib0:
-	groupadd gpib || echo "group gpib exists"
 	for i in `seq 0 15`; \
 	do \
 		mknod -m u=rw,g=rw,o= /dev/gpib$${i} c $(IBMAJOR) $${i} || exit 1; \
 		chown root:gpib /dev/gpib$${i}; \
 	done
 
-.PHONY : device-file-check
-device-file-check: /dev/gpib0
+.PHONY : device-file-check-emerge
+device-file-check-emerge: /dev/gpib0
 	@if [ ! -c /dev/gpib0 ]; then \
 		if [ -a /dev/gpib0 ]; then \
 			echo "A file or directory called /dev/gpib0 exists but it is not" \
@@ -574,10 +572,12 @@
 			"Delete your /dev/gpibX files and try again."; \
 		exit 1; \
 	fi
+	
+.PHONY : device-file-check
+device-file-check:
 
 #should move this to util/templates Makefile.am when it exists
 /etc/gpib.conf:
-	$(INSTALL_DATA) -D $(top_srcdir)/util/templates/gpib.conf /etc/gpib.conf
 
 #make sure compiled files, etc don't make it into distribution tarballs
 dist-hook:
