--- src/Base/swigpyrun.inl.old	2010-08-27 16:24:16.000000000 +0200
+++ src/Base/swigpyrun.inl	2011-12-24 00:19:25.175628049 +0100
@@ -32,7 +32,7 @@
     if (!swig_type)
         throw Base::Exception("Cannot find type information for requested type");
     
-    *ptr = SWIG_Python_NewPointerObj(obj,swig_type,own);
+    *ptr = SWIG_Python_NewPointerObj(NULL,obj,swig_type,own);
     if (*ptr == 0)
         throw Base::Exception("Cannot convert into requested type");
 
