--- calcCAP/src/cmplxmat.cpp.org	2011-02-18 02:07:06.000000000 -0800
+++ calcCAP/src/cmplxmat.cpp	2011-02-18 01:38:34.000000000 -0800
@@ -378,7 +378,7 @@
   Matrix result(d_i,d_j);
   for(int i = 0; i < d_i; i++)
     for(int j = 0; j < d_j; j++)
-      result(i,j)=::real(elem(i,j));
+      result(i,j)=elem(i,j).real(); // result(i,j)=::real(elem(i,j));
   return (result);
 }
 
@@ -387,7 +387,7 @@
   Matrix result(d_i,d_j);
   for(int i = 0; i < d_i; i++)
     for(int j = 0; j < d_j; j++)
-      result(i,j)=::imag(elem(i,j));
+      result(i,j)=elem(i,j).imag(); // result(i,j)=::imag(elem(i,j));
   return (result);
 }
 
@@ -396,7 +396,7 @@
   Matrix result(d_i,d_j);
   for(int i = 0; i < d_i; i++)
     for(int j = 0; j < d_j; j++)
-      result(i,j)=::cabs(elem(i,j));
+      result(i,j)=elem(i,j).cabs(); // result(i,j)=::cabs(elem(i,j));
   return (result);
 }
 
@@ -405,7 +405,7 @@
   CmplxMatrix result(d_i,d_j);
   for(int i = 0; i < d_i; i++)
     for(int j = 0; j < d_j; j++)
-      result(i,j)=::conjg(elem(i,j));
+      result(i,j)=elem(i,j).conjg(); // result(i,j)=::conjg(elem(i,j));
   return (result);
 }
      
