--- calcRL/src/cmplxvec.cpp.org	2011-02-18 02:25:44.000000000 -0800
+++ calcRL/src/cmplxvec.cpp	2011-02-18 01:24:09.000000000 -0800
@@ -289,7 +289,7 @@
  
  int n = d;
  CmplxVector tmp(n);
- for(int i = 0; i < n; i++)  tmp[i] = ::conjg(v[i]);
+ for(int i = 0; i < n; i++)  tmp[i] = v[i].conjg(); // tmp[i] = ::conjg(v[i]);
  return (tmp);
 }
 
@@ -297,7 +297,7 @@
  
  Vector tmp(d);
  int n = d;
- while(n--) tmp[n] = ::real(v[n]);
+ while(n--) tmp[n] = v[n].real(); // tmp[n] = ::real(v[n]);
  return (tmp);
 }
 
@@ -305,7 +305,7 @@
  
  Vector tmp(d);
  int n = d;
- while(n--) tmp[n] = ::imag(v[n]);
+ while(n--) tmp[n] = v[n].imag(); // tmp[n] = ::imag(v[n])
  return (tmp);
 }
 
@@ -313,7 +313,7 @@
 
   Vector result(d);
   for(int i = 0; i < d; i++)
-      result[i]=::cabs(v[i]);
+    result[i]=v[i].cabs(); // result[i]=::cabs(v[i]);
   return (result);
 }
 
