--- calcCAP/src/complex.h.org	2011-02-18 02:07:37.000000000 -0800
+++ calcCAP/src/complex.h	2011-02-18 01:33:02.000000000 -0800
@@ -1,10 +1,13 @@
 #ifndef _complex__h_
 #define _complex__h_
 
-#include <iostream.h>
+#include <iostream>
+using namespace std;
 
 class Complex
 {
+ public:
+
    friend Complex operator*(const double&, const Complex&);
    friend Complex operator+(const double&, const Complex&);
    friend Complex operator-(const double&, const Complex&);
@@ -16,6 +19,14 @@
    friend double     cabs(const Complex&);
    friend double     cabsm(const Complex&);
    friend Complex   conjg(const Complex&);
+
+   double     real(void);
+   double     imag(void);
+   double     arg(void);
+   double     cabs(void);
+   double     cabsm(void);
+   Complex   conjg(void);
+
    friend Complex   cmplx(const double,const double);  
    friend Complex   polar(const double&, const double&);
    friend Complex   cos(const Complex&);
@@ -37,7 +48,6 @@
    friend ostream& operator<<(ostream&, Complex&);
    friend istream& operator>>(istream&, Complex&);
 
- public:
    Complex(){}                       // constructors
    Complex(double r, double i = 0.0);
    Complex(const Complex&);
