--- calcRL/src/cmplxmat.cpp.org	2011-02-18 02:26:04.000000000 -0800
+++ calcRL/src/cmplxmat.cpp	2011-02-18 01:13:48.000000000 -0800
@@ -386,7 +386,7 @@
   Matrix result(d_i,d_j);
   for(int i = 0; i < d_i; i++)
     for(int j = 0; j < d_j; j++)
-      result(i,j)=::real(elem(i,j));
+      result(i,j)=elem(i,j).real(); // result(i,j)=::real(elem(i,j));
   return (result);
 }
 
@@ -395,7 +395,7 @@
   Matrix result(d_i,d_j);
   for(int i = 0; i < d_i; i++)
     for(int j = 0; j < d_j; j++)
-      result(i,j)=::imag(elem(i,j));
+      result(i,j)=elem(i,j).imag(); // result(i,j)=::imag(elem(i,j));
   return (result);
 }
 
@@ -404,7 +404,7 @@
   Matrix result(d_i,d_j);
   for(int i = 0; i < d_i; i++)
     for(int j = 0; j < d_j; j++)
-      result(i,j)=::cabs(elem(i,j));
+      result(i,j)=elem(i,j).cabs(); // result(i,j)=::cabj(elem(i,j));
   return (result);
 }
 
@@ -413,7 +413,7 @@
   CmplxMatrix result(d_i,d_j);
   for(int i = 0; i < d_i; i++)
     for(int j = 0; j < d_j; j++)
-      result(i,j)=::conjg(elem(i,j));
+      result(i,j)=elem(i,j).conjg(); // result(i,j)=::conjg(elem(i,j));
   return (result);
 }
      
