--- calcRL/src/calcRL.cpp.org	2011-02-18 02:24:51.000000000 -0800
+++ calcRL/src/calcRL.cpp	2011-02-18 01:30:13.000000000 -0800
@@ -43,7 +43,8 @@
 #endif
 #include <time.h>
 #include <stdio.h>
-
+#include <string.h>
+using namespace std;
 /********************** Global variables *******************/
 
    int     Nc, Die, Rec, Cir, Tri, Ell, Pol, Gnd, Tra;
@@ -564,7 +565,7 @@
 	tmp4 = Circum[i] / double(Nw);
 	tmpsum1 = 0.0;
 	tmpsum2 = 0.0;
-	sum2[i] = cmplx( 0.0, 0.0 );
+	sum2[i] = Complex( 0.0, 0.0 );
 	
 	for (j = 0; j < Nw; j ++) {
 	  
