--- calcRL/src/complex.h.org	2011-02-18 02:26:34.000000000 -0800
+++ calcRL/src/complex.h	2011-02-18 01:11:29.000000000 -0800
@@ -3,10 +3,12 @@
 
 #include <stdlib.h>
 #include <math.h>
-#include <iostream.h>
+#include <iostream>
+using namespace std;
 
 class Complex
 {
+ public:
    friend Complex operator*(const double&, const Complex&);
    friend Complex operator+(const double&, const Complex&);
    friend Complex operator-(const double&, const Complex&);
@@ -14,6 +16,14 @@
 
    friend double     real(const Complex&);
    friend double     imag(const Complex&);
+   double     real(void);
+   double     imag(void);
+   double     arg(void);
+   double     cabs(void);
+   double     cabsm(void);
+   Complex   conjg(void);
+   
+
    friend double     arg(const Complex&);
    friend double     cabs(const Complex&);
    friend double     cabsm(const Complex&);
@@ -39,7 +49,7 @@
    friend ostream& operator<<(ostream&, Complex&);
    friend istream& operator>>(istream&, Complex&);
 
- public:
+
    Complex(){}                       // constructors
    Complex(double r, double i = 0.0);
    Complex(const Complex&);
