--- calcCAP/src/calcCAP.cpp.org	2011-02-18 02:05:38.000000000 -0800
+++ calcCAP/src/calcCAP.cpp	2011-02-18 01:44:47.000000000 -0800
@@ -14,6 +14,8 @@
 #endif
 #include <time.h>
 #include <stdio.h>
+#include <string.h>
+using namespace std;
 
 /********************** Global variables *******************/
 
@@ -287,7 +289,7 @@
       
       for ( i1 = 0; i1 < Nc; i1 ++ ) {
 	
-	sumc[i1] = cmplx( 0.0, 0.0 );
+	sumc[i1] = Complex( 0.0, 0.0 );
 
 	
 	for ( j1 = 0; j1 < Nw; j1 ++ ) {
