diff -ruN digikam-2.0.0-beta1.orig/core/CMakeLists.txt digikam-2.0.0-beta1/core/CMakeLists.txt
--- digikam-2.0.0-beta1.orig/core/CMakeLists.txt	2011-01-03 15:05:12.000000000 +0100
+++ digikam-2.0.0-beta1/core/CMakeLists.txt	2011-01-30 17:39:44.000000000 +0100
@@ -1488,8 +1488,15 @@
 # ==================================================================================================
 # API documentation generation
 
+# Checks if user ask to build documentation.
+IF(BUILD_DOC)
+
 IF(DOXYGEN_FOUND)
 
+    MESSAGE(STATUS "")
+    MESSAGE(STATUS "API documentation will be installed")
+    MESSAGE(STATUS "")
+
     SET(API_DIR ${CMAKE_BINARY_DIR}/api)
     SET(SOURCE_DIR ${CMAKE_SOURCE_DIR})
     SET(DOXYFILE ${CMAKE_BINARY_DIR}/Doxyfile)
@@ -1497,6 +1504,20 @@
 
     CONFIGURE_FILE(Doxyfile.cmake ${DOXYFILE})
 
-    ADD_CUSTOM_TARGET(doc ${DOXYGEN_EXECUTABLE} ${DOXYFILE}
+    ADD_CUSTOM_TARGET(doc ALL ${DOXYGEN_EXECUTABLE} ${DOXYFILE}
                       WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+
+ELSE(DOXYGEN_FOUND)
+
+    MESSAGE(FATAL_ERROR "API documentation requested but doxygen not found")
+
 ENDIF(DOXYGEN_FOUND)
+
+ELSE(BUILD_DOC)
+
+    MESSAGE(STATUS "")
+    MESSAGE(STATUS "Documentation will NOT be installed")
+    MESSAGE(STATUS "Because the option -DBUILD_DOC=ON wasn't passed to cmake")
+    MESSAGE(STATUS "")
+
+ENDIF(BUILD_DOC)
