commit abd03787b7b1e3a392803edc5b40dfb25a69423c
Author: Gilles Caulier <caulier.gilles@gmail.com>
Date:   Mon Jul 4 13:54:40 2011 +0200

    compile with last offcial libpgf
    CCBUGS: 276969

diff --git a/cmake/modules/FindPGF.cmake b/cmake/modules/FindPGF.cmake
index b6622c4..0c82a4e 100644
--- a/cmake/modules/FindPGF.cmake
+++ b/cmake/modules/FindPGF.cmake
@@ -1,9 +1,10 @@
 # - Try to find libpgf
 # Once done this will define
 #
-#  PGF_FOUND - system has libgf
-#  PGF_INCLUDE_DIRS - the libpgf include directory
-#  PGF_LIBRARIES - Link these to use libpgf
+#  PGF_FOUND            - system has libgf
+#  PGF_INCLUDE_DIRS     - the libpgf include directory
+#  PGF_LIBRARIES        - Link these to use libpgf
+#  PGF_CODEC_VERSION_ID - PGF codec version ID.
 
 # PKG-CONFIG is required.
 INCLUDE(FindPkgConfig REQUIRED)
@@ -14,11 +15,14 @@ IF(PKG_CONFIG_FOUND)
 
     PKG_CHECK_MODULES(PGF libpgf)
 
-    MESSAGE(STATUS "PGF_INCLUDE_DIRS = ${PGF_INCLUDE_DIRS}")
-    MESSAGE(STATUS "PGF_INCLUDEDIR   = ${PGF_INCLUDEDIR}")
-    MESSAGE(STATUS "PGF_LIBRARIES    = ${PGF_LIBRARIES}")
-    MESSAGE(STATUS "PGF_LDFLAGS      = ${PGF_LDFLAGS}")
-    MESSAGE(STATUS "PGF_CFLAGS       = ${PGF_CFLAGS}")
-    MESSAGE(STATUS "PGF_VERSION      = ${PGF_VERSION}")
+    MESSAGE(STATUS "PGF_INCLUDE_DIRS     = ${PGF_INCLUDE_DIRS}")
+    MESSAGE(STATUS "PGF_INCLUDEDIR       = ${PGF_INCLUDEDIR}")
+    MESSAGE(STATUS "PGF_LIBRARIES        = ${PGF_LIBRARIES}")
+    MESSAGE(STATUS "PGF_LDFLAGS          = ${PGF_LDFLAGS}")
+    MESSAGE(STATUS "PGF_CFLAGS           = ${PGF_CFLAGS}")
+    MESSAGE(STATUS "PGF_VERSION          = ${PGF_VERSION}")
+
+    STRING(REPLACE "." "" PGF_CODEC_VERSION_ID "${PGF_VERSION}")
+    MESSAGE(STATUS "PGF_CODEC_VERSION_ID = ${PGF_CODEC_VERSION_ID}")
 
 ENDIF(PKG_CONFIG_FOUND)
diff --git a/digikam/utils/config-digikam.h.cmake b/digikam/utils/config-digikam.h.cmake
index 4ba293b..f881036 100644
--- a/digikam/utils/config-digikam.h.cmake
+++ b/digikam/utils/config-digikam.h.cmake
@@ -60,6 +60,8 @@
 /* Define to 1 if we know how to start mysqld */
 #cmakedefine HAVE_INTERNALMYSQL 1
 
+#define PGFCodecVersionID 0x${PGF_CODEC_VERSION_ID}
+
 #define LIBEXEC_INSTALL_DIR "${LIBEXEC_INSTALL_DIR}"
 
 /*
diff --git a/libs/threadimageio/pgfutils.cpp b/libs/threadimageio/pgfutils.cpp
index 758f12c..426a531 100644
--- a/libs/threadimageio/pgfutils.cpp
+++ b/libs/threadimageio/pgfutils.cpp
@@ -46,6 +46,7 @@ extern "C"
 // LibPGF includes
 
 #include "PGFimage.h"
+#include "config-digikam.h"
 
 namespace Digikam
 {
